app/src/main/java/org/oppia/android/app/devoptions/featureflags/FeatureFlagsFragmentPresenter.kt

```diff
@@ -143,7 +182,9 @@ class FeatureFlagsFragmentPresenter @Inject constructor(
       .toLiveData().observe(fragment) {
         when (it) {
           is AsyncResult.Success -> {
-            (activity as FeatureFlagsActivity).finish()
+            isRestartInitiated = true
```

Comment:
Variable naming suggestion: as to reflect restart will be needed, not already in progress.
```suggestion
            restartRequired = true
```

Reply:
updated

Reply:
Can you please double-check if this is updated?

Reply:
yes it is updated